clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/customtemplates/CustomTemplate.kt

```diff
@@ -18,6 +18,11 @@ class CustomTemplate private constructor(
      */
     val name: String,
 
+    /**
+     * The presenter associated with this template.
+     */
+    val presenter: CustomTemplatePresenter<*>,
```

Comment:
CustomTemplate seems like a data class which contains only properties for template data. Can we separate data and presenters and not add them together in an object?

Reply:
We don't serialize this class and have overridden equals and hashcode. Making it a data class and moving the presenters to an outside list somewhere would introduce complexity, that I think we can avoid for now.